tests_manual/syscall_module_path/test

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/perl
+
+use strict;
+use File::Temp qw/ tempfile /;
+use Test;
+BEGIN { plan tests => 2 }
+
+my $basedir = $0;
+$basedir =~ s|(.*)/[^/]*|$1|;
+
+###
+# functions
+
+###
+# setup
+
+# create stdout/stderr sinks
+(my $fh_out, my $stdout) = tempfile(TEMPLATE => '/tmp/audit-testsuite-out-XXXX',
+				    UNLINK => 1);
+(my $fh_err, my $stderr) = tempfile(TEMPLATE => '/tmp/audit-testsuite-err-XXXX',
+				    UNLINK => 1);
+(my $fh_out2, my $stdout2) = tempfile(
+				TEMPLATE => '/tmp/audit-testsuite-out-XXXX',
+				UNLINK => 1);
+(my $fh_err2, my $stderr2) = tempfile(
+				TEMPLATE => '/tmp/audit-testsuite-err-XXXX',
+				UNLINK => 1);
+
+###
+# tests
+
+# run the test
+my $line;
+my $uptime;
+system("cat /proc/uptime > $stdout 2> $stderr");
+while ($line = <$fh_out>) {
+	($uptime) = ($line =~ /^([0-9]*)\..*$/);
+}
+
+my $boottime = localtime(time - $uptime);
+my ($sec,$min,$hour,$mday,$mon,$year,$wday,$yday,$isdst);
+($sec,$min,$hour,$mday,$mon,$year,$wday,$yday,$isdst) = localtime(time - $uptime);
+$year += 1900; $mon += 1;
+my $starttime = "$mon/$mday/$year $hour:$min:$sec";
+print "Checking for module null PATH messages since boottime: $year:$mon:$mday $hour:$min:$sec\n";
+
+# test if we generate any audit records from the filter rule
+system("ausearch -ts $starttime -i --syscall init_module > $stdout 2> $stderr");
+system("ausearch -ts $starttime -i --syscall finit_module >> $stdout 2>> $stderr");
```

Comment:
I just had a thought, do the NULL path records come from finit_module(2)?  Since finit_module(2) doesn't actually open the file, it simply reads from a fd, could it somehow be the source of the NULL PATH records?